app/src/main/java/com/example/vybe/AddEditVibeActivity.java

```diff
@@ -101,6 +108,9 @@ protected void onCreate(Bundle savedInstanceState) {
             //TODO: set vibe and socsit dropdrowns
```

Comment:
update comment i guess but can do under refactoring once socsit dropdown is finalized